examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -26,6 +26,14 @@ spec:
         {{- if .Values.insecureSkipProxyTLSVerify }}
         - "--insecure"
         {{- end }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - all
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 9807
```

Comment:
why this specific value?
should we also set `runAsGroup`?

Reply:
Not any good reason, basically the first 4 digits of standard gravity is a UID we started to use on gravity as a default / less likely to conflict with any other users value. Most important thing is it's not root.